dev-packages/electron/electron-cli.js

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
```

Comment:
don't forget to update clients: https://github.com/theia-ide/theia/blob/master/dev-packages/application-manager/src/application-package-manager.ts#L81

Reply:
The client from your link doesn't need updating IIUC.

Reply:
Unless you want to point to `@theia/electron/electron-cli.js`?

Reply:
no, all alright :+1